apigee-test-shared/README.md

```diff
@@ -0,0 +1,25 @@
+# Apigee-test-shared
+Docker image used to run unit tests on Apigee proxies
+
+## Mounting tests
+To use this image your tests should be mounted into /code/src using this folder structure
+```
+src  
+.
+.
+.
+└───<insert code to be tested here>
+│   
+└───tests
+    |
+    └───<insert tests here>
+```
+
+## Test filename naming convention
+
+All units test filenames should be named: \<file_under_test\>.test.js 
+
+## Building the container
```

Comment:
Oh, minor issues.

 - [ ] these are instructions for running the image in a container, not building it.
 - [ ] Did you mean to include instructions for building it? :) 

🚲 Suggestion: If you add a docker-compose.yml file, you can have *it* set up the volume mounts.  Then your building and running instructions become simple calls to `docker-compose run $DC_IMG_NAME`.   And `docker-compose run` is smart enough to build the image for you if you haven't yet.

Reply:
Good point! I will change it to a build.

Reply:
Will address this